tests/pds/pds_storagesize_increase_test.go

```diff
@@ -525,7 +558,7 @@ var _ = Describe("{PerformStorageResizeBy1Gb100TimesAllDs}", func() {
 						})
 						stepLog = "Verify storage size before and after storage resize - Verify at STS, PV,PVC level"
 						Step(stepLog, func() {
-							err := ValidateDepConfigPostStorageIncrease(ds, updatedDeployment, stConfigModelUpdated, resConfigModelUpdated, initialCapacity, updatedPvcSize)
+							err := ValidateDepConfigPostStorageIncrease(ds, updatedDeployment, stConfigModelUpdated, resConfigModelUpdated, initialCapacity, updatedPvcSize, beforeResizePodAge)
```

Comment:
Whenever we have more than 5 arguments, we can send it using a struct. Which will improve modularity
here other an ds, all parameters can be put in a struct

Reply:
I agree with Arun